main.tex

```diff
@@ -15,35 +15,33 @@
+  A number of methods have been proposed to mitigate these issues. Among these, CoinJoin is an approach to structuring transactions which adds ambiguity and breaks some common assumptions that underlie heuristics used for surveillance.
+  Current implementations of CoinJoin transactions present a number of limitations which may partly explain their lack of widespread adoption.
 
-Chaumian CoinJoin~\cite{mizrahi2013blind}\cite{maxwell2013coinjoin} is a technique to facilitate untrusted construction of collaborative Bitcoin transactions, also known as CoinJoins, by utilizing Chaumian blind signatures~\cite{chaum1983blind}. However this technique requires standard denominations, which limits how such transactions can be constructed.
+  This work introduces a new protocol for centrally coordinated CoinJoin implementations utilizing keyed verification anonymous credentials and homomorphic value commitments. This is an improvement on the state of the art in both privacy and flexibility, enabling some additional use cases and reduced overheads.
+\end{abstract}
 
-We propose to switch to a Keyed-Verification Anonymous Credentials-based (KVAC) scheme~\cite{chase2019signal} to enable more flexible transaction styles, like SharedCoin and CashFusion~\cite{cashfusion} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Our generalization also enables consolidation of UTXOs, minimizing unmixed change, relaxing minimum required denominations, payments in CoinJoins, better block space efficiency, and PayJoins in CoinJoins. We call this new protocol: WabiSabi.
+\section{Introduction}
 
-\end{abstract}
+Bitcoin transactions redistribute funds by consuming as inputs the outputs of previous transactions, and creating new outputs. The protocol rules enforced by the network ensure that transactions do not arbitrarily inflate the monetary supply and that output are spent at most once. This can be simply verified because the amounts as well as the specific outputs being spent are broadcast as part of the transaction in the clear. This presents significant challenges for transacting with privacy\footnote{In this work we restrict the discussion of Bitcoin privacy to that of transactions, but there are other considerations especially at the network layer.}. % TODO cite Ron, Fistful of Bitcoins, Moser, ...?
 
-% Important remainder from removed Motivation: In the current protocol input registration links the user's inputs, as well as their change output.
+The conditions for spending an output are specified in its \texttt{scriptPubKey}, typically requiring that the spending transaction be signed by a specific public key. The signatures authorizing a transaction usually commit to the transaction in its entirety, which makes it possible for mutually distrusting parties to jointly create transactions without any risk of funds being misallocated: participants will only sign a proposed transaction after confirming that their desired outputs are included.
```

Comment:
```suggestion
The conditions for spending an output are specified in its \texttt{scriptPubKey}, typically requiring that the spending transaction to be signed by a specific public key. The signatures authorizing a transaction usually commit to the transaction in its entirety, which makes it possible for mutually distrusting parties to jointly create transactions without risk of funds being misallocated: participants will only sign a proposed transaction after confirming that their desired outputs are included.
```

Reply:
grammatically i'm not sure about this change, i think it needs a determinant of some type, but is "any" arguably too broad because risk is not eliminated entirely,... i think this needs to be rephrased a bit more